nntrainer/tensor/tensor_v2.cpp

```diff
@@ -24,7 +24,8 @@ TensorV2::TensorV2(std::string name_, Tformat fm, Tdatatype d_type) {
 #else
     throw std::invalid_argument("Error: enable-fp16 is not enabled");
 #endif
-  }
+  } else
+    throw std::invalid_argument("Error : TensorV2 itensor not initialized");
```

Comment:
Error messages should be able to let the readers correct their codes without much effort.
Provide more info about what's wrong and how to correct it.

E.g., ```Error: TensorV2 itensor cannot be initialized because the given d_type is not compatible. The supported d_types are: FP32, FP16 (if built with ENABLE_FP16)).```

Reply:
Got it. Will fix it asap